test/ambiguities.jl

```diff
@@ -17,7 +17,7 @@
         # Interims solution until we follow what was proposed in
         # https://discourse.julialang.org/t/avoid-ambiguities-with-individual-number-element-identity/62465/2
         fms = filter(x -> !any(has_type_in_signature.(x, Identity)), ms)
-        FMS_LIMIT = 22
+        FMS_LIMIT = 34
```

Comment:
Thats a lot of new ambiguities, are you sure all are necessary?

Reply:
Yes, I don't like it either but the alternative is adding a bunch of methods that will never be used outside of our tests because the are related to metric-dependent functions which dispatch down to the power manifold from power group.